src/components/thumbnail/placeholder.jsx

```diff
@@ -3,7 +3,8 @@ import React, { PropTypes } from 'react'
 const Placeholder = ({ image, size, status }) => {
     const style = {
         maxHeight: size,
-        maxWidth: size
+        maxWidth: size,
+        overflow: 'hidden'
```

Comment:
I forgot about the existing `max-width` and `max-height` styles. Overflow shouldn't be necessary.